### ISSUE TITLE
NPE occurs when it connection tired in Connection's constructor

### ISSUE THREAD

**Issue by tkrs** [tkrs/fluflu#95]

**Comment by tkrs** [comment]
```
[info] Running (fork) examples.Scheduling 10 10
[error] Exception in thread "main" java.lang.ExceptionInInitializerError
[error]         at examples.Scheduling.main(examples.scala)
[error] Caused by: java.net.ConnectException: Connection refused
[error]         at sun.nio.ch.Net.connect0(Native Method)
[error]         at sun.nio.ch.Net.connect(Net.java:454)
[error]         at sun.nio.ch.Net.connect(Net.java:446)
[error]         at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
[error]         at fluflu.Connection$ConnectionImpl.doConnect(Connection.scala:49)
[error]         at fluflu.Connection$ConnectionImpl.<init>(Connection.scala:36)
[error]         at fluflu.Connection$.apply(Connection.scala:25)
[error]         at examples.Base.<init>(examples.scala:90)
[error]         at examples.Scheduling$.<init>(examples.scala:116)
[error]         at examples.Scheduling$.<clinit>(examples.scala)
[error]         ... 1 more
```

Oh. This is already fixed...

**Comment by tkrs** [comment]
Fixed by #107
